Top/threads.c

```diff
@@ -1058,7 +1058,7 @@ int csoundSpinLockInit(spin_lock_t *spinlock) {
 
 #elif defined(MACOSX) // MacOS native locks
 
-#if MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_12
```

Comment:
more correct is this:

```
#if MAC_OS_X_VERSION_MIN_REQUIRED >= 101200
```

and this is what MacPorts does.

Reply:
Yes, I prefer numerical macros too, but for some reason not everyone is okay with them (we could not convince `Ruby` upstream, for example, and still carry our patch).